notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        (
</pre><p>Multi-line expression hard to read, consider introducing intermediate variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>